code/modules/vore/fluffstuff/custom_guns_vr.dm

```diff
@@ -483,6 +544,29 @@
 /obj/item/ammo_magazine/battlerifle/empty
 	initial_ammo = 0
 
+//---------------- M41A -----------------
```

Comment:
This all needs to go away. We already have this ammo in game and you're overriding it.